app/src/main/java/com/seanime/app/SeanimeService.kt

```
@@ -61,51 +83,40 @@ class SeanimeService : Service() {
-                            if (line.isNotBlank()) updateNotification(line.take(100))
-                        }
-                    }
-                    val exitCode = process?.waitFor()
-                    updateNotification("Process ended (Code: $exitCode)")
+                    process?.inputStream?.bufferedReader()?.use { it.forEachLine { line -> Log.d("SeanimeLog", line) } }
                 } catch (e: Exception) {
                     Log.e("SeanimeService", "Stream Error", e)
                 }
             }.start()
 
         } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)